src/app/sign-up/sign-up.component.ts

```diff
@@ -118,6 +118,11 @@ export class SignUpComponent implements OnInit, OnDestroy {
       this.router.navigate(['/', RouteNames.GET_DESO], {
         queryParams: {publicKey: this.publicKeyAdded, signedUp: true },
         queryParamsHandling: 'merge'});
+    } else if (this.globalVars.derive) {
```

Comment:
`this.globalVars.derive` is set to true in the `derive.component` but also in `app.component`. Just want to triple-check that latter value write doesn't somehow overlap with this, but I think we're fine, right?